features/smoke-tests/buyer/catalogue.feature

```diff
@@ -58,3 +58,40 @@ Scenario: User is able to search by keywords field on the search results page to
   Then I see that service.id in the search summary text
   And I see that service.id as the value of the 'q' field
   And I see that service in the search results
+
+Scenario: User is able to click on a random category
+  Given I am on the /g-cloud page
+  And I have a random g-cloud lot from the API
+  And I click that lot.name
+  Then I am on the 'Search results' page
+  And I note the number of search results
+
+  When I click a random category link
+  Then I am on the 'Search results' page
+  And I see that category_name in the search summary text
```

Comment:
Yeah I think this is justifiable: the "I click a random category link" step literally just scrapes the category name off the page, so we don't have anything approaching an object that properly represents a category. (We could put it into a dictionary, with probably just the one key, but I think that would set wrong expectations, so is less clear.)